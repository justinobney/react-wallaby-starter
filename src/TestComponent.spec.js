import React from 'react';
import {create} from 'react-test-renderer';

const TestComponent = ({test}) => <div>{test}</div>;

describe('Component: TestComponent', () => {
  xit('sample test', () => {
    const tree = create(<TestComponent test="foo" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
