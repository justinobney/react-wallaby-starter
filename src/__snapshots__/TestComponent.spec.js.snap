// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: TestComponent sample test 1`] = `
<div>
  foo
</div>
`;
